README.md

```diff
@@ -0,0 +1,8 @@
+# Python-API-request-script
+This script takes a client id and secret from a config.py file, and generates a token if the input is valid. It then sends a get request to the Beaver Bus API and prints the data from the request to the user.
+
+Version: Python 3.7.3
+
+Libraries: requests, pprint
```

Comment:
Can you list all of the dependencies to `requirements.txt` like here: https://github.com/osu-mist/locations-generator/blob/master/requirements.txt The list can be auto-generated via `pip`.

You should also document how to install them via using `pip` and how to run your code in the readme file. Example: https://github.com/osu-mist/locations-generator